docs/mobile/concepts-commands-operations.mdx

```diff
@@ -14,11 +14,7 @@ import UseCaseContainer from "../../src/components/UseCaseContainer";
 Check out the [Key Concepts](concepts-key-concepts.mdx).
 :::
 
-**Commands and operations** are runnable, independent units of work within Geocortex Mobile. Using the Geocortex Mobile SDK, it's possible to [define custom commands and operations](usecases-implement-command-operation.mdx) for use throughout the application. Commands are units of work which may take an input but do not produce an output, while operations differ because they produce output.
-
-The full list of existing commands and operations available in Geocortex Mobile can be [found in the API documentation](api-commands-operations-events.mdx).
-
-Commands and Operations are grouped by namespace, for example:
+**Commands and operations** are runnable, independent units of work within Geocortex Mobile. Using the Geocortex Mobile SDK, it's possible to [define custom commands and operations](usecases-implement-command-operation.mdx) for use throughout the application. In Geocortex Mobile, commands and operations are compatible extensions of the [ICommand interface](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/commanding). Operations are implementations of `ICommand` which have been extended to have a `name` property, allowing them to be referenced globally, and an `ExecuteAsync` method, which allows the operation to return a value. Commands and Operations are grouped by namespaces, such as `auth` or `edit`.
```

Comment:
It's not the Name (uppercase) property that allows them to be referenced globally. The Name property is actually only used for logging to write the name out. The IOperationRegistry (which is effectively a dictionary) allows for them to be referenced globally. The 'key' in the dictionary is the global name reference. Generally speaking, it is not necessary for the key to be in the value, nor is it necessary here. More generally, operations:

- are asynchronous
- are strongly typed
- can have return values

Reply:
Reworded to reflect this comment!